utility/generate_latency_graph.py

```diff
@@ -0,0 +1,88 @@
+"""
```

Comment:
Can we move this to `ceph-qe-scripts`?

Reply:
Created a new PR in ` ceph-qe-scripts ` . https://github.com/red-hat-storage/ceph-qe-scripts/pull/252